### PR TITLE
Improved UI for Juz tab and fixes

### PR DESCRIPTION
**Pull request by naveed-ahmad**

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32945

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32946

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32947

File: src/containers/AyatulKursi/index.js [hunk 1]
```diff
@@ -50,9 +50,10 @@ const TopOptions = Loadable({
 });
 
 const title = 'Ayatul Kursi';
-const description = 'Ayatul Kursi is verse 255 of the second chapter, ' +
-                    'Surah al-Baqarah (The Chapter of the Cow), in the ' +
-                    'Holy Quran. It is also known as the Throne Verse.';
+const description =
```
Comment (sabeurthabti) [inline src/containers/AyatulKursi/index.js]: why not template literals?

Comment (naveed-ahmad) [reply src/containers/AyatulKursi/index.js]: Sure,  next PR maybe. These changes are from lint auto fixer :)

**Pull request merged**